impeller/entity/entity_pass.cc

```diff
@@ -537,6 +537,7 @@ EntityPass::EntityResult EntityPass::GetEntityForElement(
       // The subpass will need to read from the current pass texture when
       // rendering the backdrop, so if there's an active pass, end it prior to
       // rendering the subpass.
+      pass_context.GetRenderPass(pass_depth);
```

Comment:
I'm not sure I fully understand this case 🤔

Reply:
If the very first thing we render in an EntityPass is a subpass that happens to have a backdrop filter, than that backdrop filter will end up rendering the raw, uncleared texture that came straight out of the texture cache. By calling `pass_context.GetRenderPass` here, we force at least one pass to get created with the correct config before that sampling happens, which will clear the texture.

Added this to a comment inline.

Reply:
Oh, I see - so we're attaching the texture but we never created any rendering commands to record to it, so we never cleared the values. hmm. Slight flaw in the render target cache design